Source/WebCore/bindings/js/JSDOMConvertStrings.h

```diff
@@ -28,16 +28,22 @@
 #include "IDLTypes.h"
 #include "JSDOMConvertBase.h"
 #include "StringAdaptors.h"
+#include "TrustedType.h"
 
 namespace WebCore {
 
+class ScriptExecutionContext;
+
+enum class ShouldConvertNullToEmptyString : bool { No, Yes };
+
 WEBCORE_EXPORT String identifierToString(JSC::JSGlobalObject&, const JSC::Identifier&);
 WEBCORE_EXPORT String identifierToByteString(JSC::JSGlobalObject&, const JSC::Identifier&);
 WEBCORE_EXPORT String valueToByteString(JSC::JSGlobalObject&, JSC::JSValue);
 WEBCORE_EXPORT AtomString valueToByteAtomString(JSC::JSGlobalObject&, JSC::JSValue);
 WEBCORE_EXPORT String identifierToUSVString(JSC::JSGlobalObject&, const JSC::Identifier&);
 WEBCORE_EXPORT String valueToUSVString(JSC::JSGlobalObject&, JSC::JSValue);
 WEBCORE_EXPORT AtomString valueToUSVAtomString(JSC::JSGlobalObject&, JSC::JSValue);
+String getTrustedTypeCompliantString(TrustedType, JSC::JSGlobalObject&, JSC::JSValue, const String& sink, ShouldConvertNullToEmptyString);
```

Comment:
WebKit coding style names this function without the word “get”.

Reply:
Ah okay, fixed that.